services/frontend-service/src/ui/components/ReleaseDialog/ReleaseDialog.scss

```diff
@@ -2,60 +2,163 @@
 @use '@material/card/mdc-card';
 
 .release-dialog {
-    border-radius: 15px 15px 0px 0px;
-    background-color: transparent;
-    border-radius: 10px;
+  border-radius: 15px 15px 0px 0px;
```

Comment:
please attach a screenshot to the PR of how it looks now.

Reply:
![Screenshot 2022-12-19 at 15 45 16](https://user-images.githubusercontent.com/113351170/208464877-d0f313d6-78bb-41c2-bcd9-5f8c74d8f7ef.png)